qiita_pet/templates/metadata_summary.html

```diff
@@ -5,6 +5,10 @@
 <h1>Metadata Summary for Study {{study_title}}</h1>
 
 <div>
+    <a class="glyphicon glyphicon-arrow-left" href="{{back_button_path}}">Back</a>
```

Comment:
Can you add the button class to this tag? and word-spacing so there's some space between the icon and the word? 


Like this:

```
 <a class="btn btn-default glyphicon glyphicon-....." href="...." style="word-spacing: -10px;">....</a>
```

See this [line](https://github.com/biocore/qiita/blob/master/qiita_pet/templates/study_description_templates/prep_template_panel.html#L68) for an exampl.